ai2thor/tests/test_unity.py

```diff
@@ -80,6 +80,25 @@ def test_small_aspect():
     event = controller.step(dict(action='Initialize', gridSize=0.25))
     assert event.frame.shape == (64, 128, 3)
 
+@pytest.mark.parametrize("controller", [fifo_controller])
+def test_fast_emit(controller):
+    event = controller.step(dict(action='RotateRight'))
+    event_fast_emit = controller.step(dict(action='TestFastEmit', rvalue='foo'))
+    # assert that when actionFastEmit is off that the objects are different
+    assert id(event.metadata['objects']) !=  id(event_fast_emit.metadata['objects'])
+
+    fast_controller = UnityTestController(server_class=FifoServer, fastActionEmit=True)
```

Comment:
May be better to separate this into a second test, as it is testing different stuff and not using pytest's parametrization of the `controller` variable.

Reply:
agreed. refactored into a separate test.